packages/utilities/fast-gestures/docs/api-report.md

```diff
@@ -0,0 +1,279 @@
+
+```ts
+
+// @public (undocumented)
+export abstract class Disposable implements IDisposable {
+    // (undocumented)
+    dispose(): void;
+    // (undocumented)
+    static None: Readonly<IDisposable>;
+    // (undocumented)
+    protected _register<T extends IDisposable>(t: T): T;
+    // (undocumented)
+    protected get toDispose(): IDisposable[];
+    // (undocumented)
+    protected _toDispose: IDisposable[];
+}
+
+// @public (undocumented)
+export interface IDisposable {
+    // (undocumented)
+    dispose(): void;
+}
+
+// @public (undocumented)
+export type IPointerGestureConfig = Partial<Record<PointerGestureFeature, boolean>>;
+
+// @public (undocumented)
+export interface IPointerGestureInfo {
+    // (undocumented)
+    pointerInfo: IPointerInfo;
+    // (undocumented)
+    target?: PointerTarget;
+}
+
+// @public (undocumented)
+export interface IPointerGestureOptions {
+    // (undocumented)
+    coordinateType?: PointerCoordinateType;
+    // (undocumented)
+    doubleTapTimeout?: number;
+    // (undocumented)
+    isHoverEnabled?: boolean;
+    // (undocumented)
+    isMultiTouchEnabled?: boolean;
+    // (undocumented)
+    longPressDuration?: number;
+    // (undocumented)
+    mouseMoveThreshold?: number;
+    // (undocumented)
+    pinchDistanceThreshold?: number;
+    // (undocumented)
+    pressThreshold?: number;
+    // (undocumented)
+    rotationAngleThreshold?: number;
+    // (undocumented)
+    shouldSetFocusOnActiveElement?: boolean;
+    // (undocumented)
+    singleTapTime?: number;
+    // (undocumented)
+    swipeThresholdX?: number;
+    // (undocumented)
+    swipeThresholdY?: number;
+    // (undocumented)
+    swipeVelocityThreshold?: number;
+    // (undocumented)
+    touchMoveThreshold?: number;
+    // (undocumented)
+    translationDirection?: PointerTranslationDirection;
+}
+
+// @public (undocumented)
+export interface IPointerInfo extends Partial<PointerEvent> {
+    eventType: string;
+    gestureType?: PointerGesture;
+    height: number;
+    id: number;
+    initialPinchDistance: number;
+    initialPressTime: number;
+    initialRotationAngle: number;
+    isDown: boolean;
+    isLongPressComplete: boolean;
+    lastUpdateTime: number;
+    longPressProgress: number;
+    movementX: number;
+    movementY: number;
+    offsetX: number;
+    offsetY: number;
+    pinchDistance: number;
+    rotationAngle: number;
+    rotationDirection: PointerRotationDirection;
+    screenX: number;
+    screenXDelta: number;
+    screenXInitial: number;
+    screenY: number;
+    screenYDelta: number;
+    screenYInitial: number;
+    tangentialPressure: number;
+    tiltX: number;
+    tiltY: number;
+    twist: number;
+    width: number;
+    x: number;
+    xDelta: number;
+    xInitial: number;
+    xVelocity: number;
+    y: number;
+    yDelta: number;
+    yInitial: number;
+    yVelocity: number;
+}
+
+// @public (undocumented)
```

Comment:
Can you mark this interface (and any other export that not intended to be a public export) with the `@internal` tsdoc tag so that this doesn't get emitted to the api report?